pages/ProductPage.ts

```diff
@@ -37,19 +37,21 @@ export class ProductPage {
       }
 
       async getFirstProductName(): Promise<string> {
-        return await this.firstProductNameInList.innerText();
+        return ((await this.firstProductNameInList.innerText()).trim());
```

Comment:
I'd divide this string into two lines

```suggestion
const productNameText = await this.firstProductNameInList.innerText()
        return productNameText.trim();
```

The same below